napari/_qt/qt_viewer.py

```diff
@@ -312,7 +312,7 @@ def _add_layer(self, event):
         """
         layers = event.source
         layer = event.item
-        vispy_layer = create_vispy_visual(layer)
+        vispy_layer = create_vispy_visual(layer, self.viewer.camera)
```

Comment:
```suggestion
        vispy_layer = create_vispy_visual(layer)
        self.viewer.camera.events.center.connect(vispy_layer._on_camera_move)
```
See comment below, I would have taken this approach. It's not a huge deal, but curious your thoughts on the relative merits

Reply:
Yeah I push a change to this way.